src/Phinx/Console/Command/SeedCreate.php

```diff
@@ -198,7 +198,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $namespace = $config instanceof NamespaceAwareInterface ? $config->getSeedNamespaceByPath($path) : null;
         $classes = [
-            '$namespaceDefinition' => $namespace !== null ? ('namespace ' . $namespace . ';') : '',
+            '$namespaceDefinition' => $namespace !== null ? (PHP_EOL . 'namespace ' . $namespace . ';' . PHP_EOL) : '',
```

Comment:
Updating this line so that it looks and works the same as the similar line we have for migrations:

https://github.com/cakephp/phinx/blob/09d8f17f782bfb4463e0c5282fd6ef4953a6e13b/src/Phinx/Console/Command/Create.php#L292